Vertical interpolation in View menu

**Describe the bug**
After loading a raw wrfout file, I'd like to view geopotential height on 500 hPa. I know I need to interpolate the variable onto 500 hPa. I checked the Interpolate Vertical Level box, picked the hydrostatic pressure as the vertical variable and enter 500 in the Desired Level box. Then I click the Interpolate button. Got the following error:

plugins/gis4wrf/core/transforms/wrf_netcdf_to_gdal.py", line 162, in convert_wrf_nc_var_to_gdal_dataset
              assert len(dims) == len(shape), f'|{dims}| != |{shape}|'
             AssertionError: |('Time', 'south_north', 'west_east')| != |(399, 399)|

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**System Information (please complete the following information):**
 - OS: Linux (ubuntu)
 - QGIS: 3.10.8-A Coruña
 - gis4wrf: 0.14.4
![Screenshot from 2021-02-01 22-26-43](https://user-images.githubusercontent.com/20599766/106561146-06d36780-64dd-11eb-803b-be612197b510.png)


**Additional context**
Add any other context about the problem here.

--- a/gis4wrf/core/transforms/wrf_netcdf_to_gdal.py
+++ b/gis4wrf/core/transforms/wrf_netcdf_to_gdal.py
@@ -149,7 +149,7 @@ def convert_wrf_nc_var_to_gdal_dataset(
                 var = wrf.getvar(ds, var_name, timeidx=wrf.ALL_TIMES, squeeze=False, meta=False)
             if interp_level is not None:
                 vert = wrf.getvar(ds, interp_vert_name, timeidx=wrf.ALL_TIMES, squeeze=False, meta=False)
-                var = wrf.interplevel(var, vert, interp_level, missing=no_data, meta=False)
+                var = wrf.interplevel(var, vert, interp_level, missing=no_data, squeeze=False, meta=False)
                 dims = MASS
             else:
                 dims = DIAG_DIMS[var_name]

